Poker/Game.cs

```diff
@@ -14,29 +14,24 @@ class Game
         string Player_1 = "Player 1";
         string Player_2 = "Player 2";
 
-        List<Card> Hand_1 = new List<Card>();
-        List<Card> Hand_2 = new List<Card>();
-
         public Game()
         {
             Players = new List<Player>();
 
             Players.Add(new Player(Player_1));
             Players.Add(new Player(Player_2));
-             
+
             Deck deck = new Deck();
-            
+
         }
 
-        public void DealCards(List<Card> listOfCards, List<Card> hand)
+        public void DealCards()
         {
-            foreach (Player item in Players)
+            foreach (Player player in Players)
```

Comment:
Avoid using the word item in your foreach loops. Something like player makes more sense